libkbfs/interfaces.go

```diff
@@ -365,7 +371,9 @@ type KBPKI interface {
 	// HasVerifyingKey returns nil if the given user has the given
 	// VerifyingKey, and an error otherwise.
 	HasVerifyingKey(ctx context.Context, uid keybase1.UID,
-		verifyingKey VerifyingKey, atServerTime time.Time) error
+		verifyingKey VerifyingKey, atServerTime time.Time,
+	        checkUnverified bool) error
```

Comment:
I have a slight preference for a separate interface method for checking unverified keys, to make it really hard to do it by accident.  Though the use of this bool in the code is clear (it's not just a `true`, it's a named var), so I'm ok with this if you prefer it.